src/components/DayPicker.jsx

```diff
@@ -629,14 +618,14 @@ class DayPicker extends React.Component {
     const newMonth = currentMonth.clone();
     const firstDayOfWeek = this.getFirstDayOfWeek();
     if (monthTransition === PREV_TRANSITION) {
-      if (onPrevMonthClick) onPrevMonthClick();
       newMonth.subtract(1, 'month');
+      if (onPrevMonthClick) onPrevMonthClick(newMonth);
```

Comment:
This is exactly what I was going to suggest. :)

Reply:
haha nice!